### ISSUE TITLE
White bg color showing behind the fixed main nav

### ISSUE THREAD

**Issue by arunsasi** [dneukirchen/joomla-org-template#10]
When user scrolls downs and scroll up fast a white bg color is showing behind the fixed main nav

**Comment by aDaneInSpain** [comment]
I think this was already reported and there is no easy solution to this. @dneukirchen?

**Comment by dneukirchen** [comment]
No, there is not. I need some time to test some possible workarounds.
